mocca-client/src/main/java/com/paypal/mocca/client/OperationType.java

```diff
@@ -0,0 +1,32 @@
+package com.paypal.mocca.client;
+
+import java.lang.annotation.Annotation;
+
+/**
+ * Enumeration representing each supported GraphQL operation.
+ * This enumeration is internal purposes only, bot being
+ * part of Mocca API.
+ *
+ * @author fabiocarvalho777@gmail.com
+ */
+enum OperationType {
+
+    Query("query"),
+    Mutation("mutation");
+
+    private final String value;
+
+    OperationType(String value) {
+        this.value = value;
+    }
+
+    static OperationType getFromAnnotation(Annotation operationAnnotation) {
```

Comment:
I always copy the good stuff when I can.  Copying `java.lang.Boolean#valueOf`...
```suggestion
    static OperationType valueOf(final Annotation operationAnnotation) {
```